sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.hpp

```diff
@@ -283,6 +299,8 @@ struct _ur_device_handle_t : _pi_object {
   // Return whether to use immediate commandlists for this device.
   ImmCmdlistMode useImmediateCommandLists();
 
+  EventsScope eventsScope = AllHostVisible;

```

Comment:
Coding style is to start var names with an upper case and functions names with a lower case.

```suggestion
  EventsScope EventsScope = AllHostVisible;

```

Also please add a comment about what this new member does.

Reply:
@smaslov-intel : cool, adding it.